src/components/Transactions/Transactions.js

```diff
@@ -56,115 +61,120 @@ const TransactionsSkeleton = () => (
   </Card>
 );
 
-const Transactions = ({
-  transactions,
-  toggleTransactionsList,
-  isTransactionsItemsLoading,
-  isExpanded,
-  isShowMoreButtonHidden,
-  error,
-  isTransactionsSectionLoading,
-  t
-}) =>
-  isTransactionsSectionLoading ? (
-    <TransactionsSkeleton />
-  ) : (
-    <WrapStyled>
-      {error.length !== 0 ? (
+const Transactions = ({ t }) => {
+  const {
+    transactions: transactionsList,
+    showToggleButton,
+    error: transactionsError,
+    loading: isTransactionsSectionLoading,
+    isListExpanded: isTransactionListExpanded
+  } = useSelector(state => state.transactions);
+
+  const transactionsToShow = isTransactionListExpanded
+    ? transactionsList
+    : transactionsList.slice(0, DEFAULT_TRANSACTIONS_NUMBER);
+
+  const dispatch = useDispatch();
+
+  const toggleTransactionsList = () => dispatch(toggleTransactionList());
```

Comment:
As this function is used only in one place we can remove it and simply use dispatch(toggleTransactionList()) in Transactions.js l:159